composer.json

```diff
@@ -21,10 +21,13 @@
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
         "doctrine/phpcr-odm": "^1.4",
+        "jackalope/jackalope": "^1.0",
+        "jackalope/jackalope-doctrine-dbal": "^1.0",
+        "jackalope/jackalope-jackrabbit": "^1.0",
         "symfony/symfony": "^4.4 || ^5.4 || ^6.0",
         "symfony/monolog-bundle": "^2.0 || ^3.0",
         "symfony/phpunit-bridge": "^5.4 || ^6.0",
-        "symfony-cmf/testing": "^2.1 || ^3.0 || ^4.0"
+        "symfony-cmf/testing": "dev-master as 4.5.0"
```

Comment:
@dbu I think we could tag the merged https://github.com/symfony-cmf/Testing/pull/213 so I can remove here `dev-master`?

Reply:
sorry, this went under. i created the tag now: https://github.com/symfony-cmf/Testing/releases/tag/4.4.2